src/main/java/org/nuxeo/ecm/platform/groups/audit/seam/ExportGroupManagementActions.java

```diff
@@ -36,6 +36,7 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.poi.util.IOUtils;
```

Comment:
Any advantage over our FileUtils ? Also we probably now have an unused import...

Reply:
FileUtils is now a "Wrapper" to call IOUtils

Reply:
But don't use the `IOUtils` from Apache POI. Use `org.apache.commons.io.IOUtils`.